bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/i18n/mielecloud_de.properties

```diff
@@ -238,6 +238,12 @@ channel-type.mielecloud.door_state.description=Gibt an, ob die Tür des Geräts
 channel-type.mielecloud.door_alarm.label=Türalarm
 channel-type.mielecloud.door_alarm.description=Gibt an, ob der Türalarm des Geräts aktiv ist.
 
+channel-type.mielecloud.water_consumption_current.label=Aktueller Wasserverbrauch
+channel-type.mielecloud.water_consumption_current.description=Durch das derzeit laufende Programm verbrauchte Wasser seit Programmstart.
+
+channel-type.mielecloud.energy_consumption_current.label=Aktueller Stromverbrauch
+channel-type.mielecloud.energy_consumption_current.description=Durch das derzeit laufende Programm verbrauchter Strom seit Programmstart.
+
```

Comment:
Translations have to be provided in Crowdin, see https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations.

Reply:
Will do so, but what does this mean for the translation files? Should I leave it as is now, remove the file or remove the new entries?

Reply:
Only remove the new entries, that will avoid collisions with Crowdin being upstream. The new entries can then be added in Crowdin when the PR is merged, and the proof-reading process will be utilized there.